src/components/Calculator.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, useCalculator } from '@/components';
+import { ControlAction, Digit, Operation } from '@/types';
+
+import styles from './Calculator.module.css';
+
+const DIGITS = [
```

Comment:
이렇게 작성해볼 수 있지 않을까요? 만약 `toReversed`가 지원되지 않는 환경이라면 `reverse`를 사용해도 괜찮을 거 같기도 하구요.

```suggestion
const DIGITS = Object.values(Digit).toReversed()
```

Reply:
> 이렇게 작성해볼 수 있지 않을까요? 만약 `toReversed`가 지원되지 않는 환경이라면 `reverse`를 사용해도 괜찮을 거 같기도 하구요.

제안주신 내용과 같은 Enum values를 `reverse`로 변환하는 것에 대한 고민이 됐었습니다.
reverse로 변환한다면, 간결하게 표현할 수 있다고 느꼈어요.

그러면서 `가독성` `유지보수` 측면을 한번 더 고려해보게 됐습니다.
- 코드를 확인하는 입장에선 `Digit` 객체 값을 한번 더 확인해야 할 상황이 있을수 있겠다.
- 업데이트 되는 상황이 생긴다면 `변경점`을 고려해야 할 수 있겠다.
  - 사소하지만 '00' 같은 값이 업데이트 됐고, 순서가 달라지면 신경쓰일수 있겠다.

이런 부분들에서 9~0까지는 직접 명시적으로 작성해도 되지 않을까? 로 결론에 이르게 됐습니다. 🙂

Reply:
넵 그 방향도 일리 있어 보입니다~ 여기는 일단 그대로 유지하시죠!